src/neo/ProtocolSettings.cs

```diff
@@ -33,10 +33,18 @@ public static ProtocolSettings Default
             {
                 if (_default == null)
                 {
-                    var configuration = Utility.LoadConfig("protocol");
+                    IConfigurationRoot configuration;
+                    try
+                    {
+                        configuration = Utility.LoadConfig("protocol");
+                    }
+                    catch (Exception e)
+                    {
+                        Utility.Log(nameof(ProtocolSettings), LogLevel.Error, "Failed parsing protocol.json. Using default settings. Error: " + e.Message);
+                        configuration = new ConfigurationBuilder().Build();
+                    }
```

Comment:
We can move the `try...catch` to `Utility.LoadConfig()`, then it can be used in other places.

Reply:
👌